src/openms/source/FORMAT/ControlledVocabulary.cpp

```diff
@@ -382,7 +396,8 @@ namespace OpenMS
             term.xref_type = CVTerm::XSD_STRING;
             continue;
           }
-          if (line_wo_spaces.hasSubstring("xsd:integer") || line_wo_spaces.hasSubstring("value-type:xsd:int"))
+          if (line_wo_spaces.hasSubstring("xsd:integer") 
+          || line_wo_spaces.hasSubstring("xsd:int"))
```

Comment:
Why the change here? I agree, yours looks more correct but maybe this really somehow has to be valuetype:xsd:int

Reply:
This was a copy and paste bug that I introduced before in the previous PR (support for relationship has_value_type inner obo) and now is fixed.